### DISCUSSION TITLE
这个地图有点问题

### DISCUSSION THREAD

**Discussion by DANaini13** [WeNeedHome/SummaryOfLoanSuspension#664] [General]
西藏的边界不要用虚线

**Comment by yalo-ddaa** [comment]
建议你自己画一个，然后提交PR合并过来。有这份心就多为国家做点实事。

**Reply by DANaini13** [reply]
我可以去画一个提交PR，我觉得你的建议没问题啊。

**Comment by trunklater** [comment]
这个谷歌地图确实有问题，建议改下。

**Comment by trunklater** [comment]
还是建议先把地图热点那个下了。这个谷歌地图就是缺失了。免得被别有用心的人利用了。

**Comment by hdp521** [comment]
地图都下了 
那这个事就这样了，别维权了，你没权利，一个地图都这么计较

**Reply by DANaini13** [reply]
不啊，计较地图，跟这件事情是两回事。你不在意可以选择不发言，我计较地图应该不耽误你维权吧。怎么？让你心情变差了吗？

**Comment by killdayu** [comment]
感谢提醒，目前已经删除了
